[2022-06-20 07:03:57,419] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: DM-COUNT-account-turnover.START_COUNT_BY_DAY_3 scheduled__2022-06-18T21:27:30.430782+00:00 [queued]>
[2022-06-20 07:03:57,451] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: DM-COUNT-account-turnover.START_COUNT_BY_DAY_3 scheduled__2022-06-18T21:27:30.430782+00:00 [queued]>
[2022-06-20 07:03:57,451] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-06-20 07:03:57,451] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-06-20 07:03:57,451] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-06-20 07:03:57,506] {taskinstance.py:1259} INFO - Executing <Task(PostgresOperator): START_COUNT_BY_DAY_3> on 2022-06-18 21:27:30.430782+00:00
[2022-06-20 07:03:57,514] {standard_task_runner.py:52} INFO - Started process 190 to run task
[2022-06-20 07:03:57,524] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'DM-COUNT-account-turnover', 'START_COUNT_BY_DAY_3', 'scheduled__2022-06-18T21:27:30.430782+00:00', '--job-id', '427', '--raw', '--subdir', 'DAGS_FOLDER/dm_fill_account_turnover.py', '--cfg-path', '/tmp/tmpqbl1f9jh', '--error-file', '/tmp/tmpv3_pkevb']
[2022-06-20 07:03:57,524] {standard_task_runner.py:77} INFO - Job 427: Subtask START_COUNT_BY_DAY_3
[2022-06-20 07:03:57,658] {logging_mixin.py:109} INFO - Running <TaskInstance: DM-COUNT-account-turnover.START_COUNT_BY_DAY_3 scheduled__2022-06-18T21:27:30.430782+00:00 [running]> on host c087b85f9f15
[2022-06-20 07:03:57,758] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=DM-COUNT-account-turnover
AIRFLOW_CTX_TASK_ID=START_COUNT_BY_DAY_3
AIRFLOW_CTX_EXECUTION_DATE=2022-06-18T21:27:30.430782+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-18T21:27:30.430782+00:00
[2022-06-20 07:03:57,777] {base.py:79} INFO - Using connection to: id: postgres. Host: postgres, Port: 5432, Schema: postgres, Login: ***, Password: ***, extra: {}
[2022-06-20 07:03:57,794] {dbapi.py:225} INFO - Running statement: CALL ds.fill_account_turnover_f (%(date)s::date);, parameters: {'date': '2018-01-3'}
[2022-06-20 07:03:57,917] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=DM-COUNT-account-turnover, task_id=START_COUNT_BY_DAY_3, execution_date=20220618T212730, start_date=20220620T070357, end_date=20220620T070357
[2022-06-20 07:03:57,954] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-06-20 07:03:58,051] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
